### PR TITLE
docs(cip-6): update the reference implementation

### PR DESCRIPTION
**Pull request by ninabarbakadze**
## Overview

Fixes - #112

### CODE REVIEW

**Comment by jcstein** [comment]
let's cover move from last call to final in a new PR, as I think it is out of scope for the issue

**Comment by ninabarbakadze** [comment]
> let's cover move from last call to final in a new PR, as I think it is out of scope for the issue

reverting the change

**Review by jcstein — APPROVED** [review]

File: cips/cip-6.md [hunk 1]
```diff
@@ -81,9 +81,9 @@ if err != nil {
 
 This will now reject undecodable transactions. Decodable transactions will still have to pass the `AnteHandlers` before being accepted in a block so no further change is required.
 
-A mechanism to enforce a minimum fee is already in place in the `DeductFeeDecorator`. Currently, this is the validators locally set value sourced from their config. To switch to a network-wide value, we introduce a new `param.Subspace` which solely consisty of the min gas price. If this is unpopulated, it will be set to the default value of `0.002utia` (which matches the current local min gas price). As a `param.Subspace`, this value can be modified via governance as can any other parameter.
+A mechanism to enforce a minimum fee is already in place in the `DeductFeeDecorator`. Currently, this is the validators locally set value sourced from their config. To switch to a network-wide value, we introduce a new `param.Subspace`, `minfee`, which solely consists of the global min gas price. If this is unpopulated, it will be set to the default value of `0.002utia` (which matches the current local min gas price). As a `param.Subspace`, this value can be modified via governance as can any other parameter.
```
Comment (rootulp) [inline cips/cip-6.md:84]: > To switch to a network-wide value, we introduce a new `param.Subspace`, `minfee`, which solely consists of the global min gas price.

AFAIK the decorator isn't _switching_ but it's adding an additional rule. The old rule that uses the validators locally set config still applies. How about something like:

```suggestion
A mechanism to enforce a minimum fee is already in place in the `DeductFeeDecorator`. Currently, the decorator uses a  min gas price sourced from the validator's local config. To introduce a network-wide constraint on min gas price, we introduce a new `param.Subspace=minfee`, which contains the global min gas price. If the param is unpopulated, it defaults to `0.002utia` (which matches the current local min gas price).
```

Comment (ninabarbakadze) [reply cips/cip-6.md]: sounds good to me

**Review by rootulp — APPROVED** [review]

**Pull request merged**